alembic/versions/202501180001_tighten_nutrition_constraints.py

```
@@ -0,0 +1,68 @@
+"""Tighten nutritional constraints for recipes and meals.
+
+Revision ID: 202501180001
+Revises: 202501170001
+Create Date: 2025-01-18
+"""
+
+from __future__ import annotations
+
+import sqlalchemy as sa
```

Flake8 (7.3.0)

[error] 'sqlalchemy as sa' imported but unused

(F401)